gateway-js/src/__tests__/integration/configuration.test.ts

```diff
@@ -301,6 +320,25 @@ describe('gateway config / env behavior', () => {
   });
 
   describe('schema config delivery endpoint configuration', () => {
```

Comment:
Can we add a test for 2+ comma separated endpoints?

Reply:
I added two more end-points to that test.